hack/test-cmd.sh

```diff
@@ -4,12 +4,15 @@
 # simple scenarios.  It does not require Docker so it can run in travis.
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 os::util::environment::setup_time_vars
+source "$(dirname "${BASH_SOURCE}")/local-up-master/lib.sh"
```

Comment:
One of the points of the `init.sh` is to give you `$OS_ROOT` ... why not use it?